resources/definitions/vzbot_custom.def.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "VzBot Custom",
```

Comment:
What is the point of having this machine in there? It doesn't make a lot of sense to me.

Reply:
There are modified versions that have a different bed size. So the Platform STL file would not fit. Just like the voron2_custom.